docs/api-reference/global-functions/Date.md

```diff
@@ -1,17 +1,23 @@
 ---
-description: Date Date(Date p0)
+uid: crmscript_global_date
+title: Date
+description: CRMScript Date constructor
 intellisense: Void.Date
 langref: 1
-keywords: Date(Date)
+keywords: Date(Date), Date(String)
 so.topic: reference
 ---
 
```

Comment:
This overall description does not seem to follow the same conventions established in previous docs.

Append to description: The default value is now.

Perhaps link to, or include examples from [Data types documentation](https://github.com/SuperOfficeDocs/crmscript/blob/main/docs/datatypes/date-type.md).